packages/patternfly-4/react-integration/cypress/integration/button.spec.ts

```diff
@@ -0,0 +1,31 @@
+describe('Button Demo Test', () => {
+  it('Navigate to demo section', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('#button-demo-nav-item-link').click();
+    cy.url().should('eq', 'http://localhost:3000/button-demo-nav-link')
+  });
+
+  it('Verify primary button', () => {
+    cy.get('button').should('have.class', 'pf-m-primary')
+  }); 
+
+  it('Verify primary button clicks', () => {
+    cy.get('.pf-m-primary').click()
```

Comment:
You should update this test to do something on click that you can verify.  Maybe show a label?